sources/osgShadow/ShadowTextureAtlas.js

```diff
@@ -166,7 +149,7 @@ ShadowTextureAtlas.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectI
         // Texture stuff: call parent class method
         Texture.prototype.apply.call( this, state, texUnit );
 
-        if ( this._lightUnit === -1 )
+        if ( this._lightNumber === -1 )
```

Comment:
it does not make sense this._lightNumber does not exist, it will always fails the test